### DISCUSSION TITLE
posts/the-map-linear-gd/

### DISCUSSION THREAD

**Discussion by giscus** [MoldyMacaroniX/gd-website#57] [Comments]
# posts/the-map-linear-gd/

Geometry Dash 2.2 seems like it's just around the corner but also so far away at the same time. But one feature that especially makes the wait hard is the "Map," a linear world with levels made by RobTop.

https://www.dashword.net/posts/the-map-linear-gd/

**Comment by clustifle** [comment]
my video :D

**Reply by MikaKC** [reply]
Yeah it’s the only video I found on the web as the discord video was invalid :). Thanks for putting this video!

**Comment by libracrumbs** [comment]
> He also revealed that The Map would be in the creator menu. I can't be the only one who thinks this is a very interesting choice. Why is this here? There might be a creator contest similar to the gauntlets where levels will compete to be in it. It could even get short challenges rated! It would be hilarious to see VSC as the final level.

Noo don't give RobTop ideas, he'll actually do it
